.travis.yml

```diff
@@ -44,7 +44,7 @@ matrix:
         - TESTMODE=fast
         - COVERAGE=
         - NPY_RELAXED_STRIDES_CHECKING=1
-        - NUMPYSPEC="git+git://github.com/numpy/numpy.git@v1.9.1"
+        - NUMPYSPEC="--upgrade git+git://github.com/numpy/numpy.git@v1.9.1"
```

Comment:
This is going to quietly break again as soon as numpy 1.10 is out. Or maybe it is already broken, if preinstalled numpy is 1.9.2...